src/main/java/com/treasuredata/client/TDHttpClient.java

```diff
@@ -371,7 +372,7 @@ public RequestContext withRootCause(TDClientException e)
     }
 
     protected <Result> Result submitRequest(RequestContext context, TDHttpRequestHandler<Result> handler)
-            throws TDClientException, InterruptedException
+            throws TDClientException, InterruptedException, InterruptedIOException
```

Comment:
`InterruptedIOException` can be removed because this function does not throw InterruptedIOException.

Reply:
It's needed; otherwise, the code does not compile. Expand the code in the PR and you'll see that it is not the method containing the catch clauses.

Reply:
Still I think the submitRequest won't throw InterruptedIOException, it is a checked exception, and this PR does not have new logic to throw checked exception.  
Actually all exceptions are cached here inside submitRequest:   https://github.com/treasure-data/td-client-java/blob/a532fdd965379e8e2a60e70cf48f56c2a7363a7b/src/main/java/com/treasuredata/client/TDHttpClient.java#L433

Reply:
The exception is rethrown from within the catch block
```
com.treasuredata.client.TDClientProcessingException: [EXECUTION_FAILURE] interrupted The root cause: interrupted
2023-01-31T23:36:21.549292135Z 	at com.treasuredata.client.TDRequestErrorHandler.defaultExceptionResolver(TDRequestErrorHandler.java:209)
2023-01-31T23:36:21.549301218Z 	at com.treasuredata.client.TDRequestErrorHandler.defaultErrorResolver(TDRequestErrorHandler.java:145)
2023-01-31T23:36:21.549308635Z 	at com.treasuredata.client.TDHttpRequestHandler.resolveError(TDHttpRequestHandler.java:88)
2023-01-31T23:36:21.549316343Z 	at com.treasuredata.client.TDHttpClient.submitRequest(TDHttpClient.java:435)
2023-01-31T23:36:21.549324135Z 	at com.treasuredata.client.TDHttpClient.submitRequest(TDHttpClient.java:475)
2023-01-31T23:36:21.549339177Z 	at com.treasuredata.client.TDHttpClient.call(TDHttpClient.java:566)
2023-01-31T23:36:21.549406010Z 	at com.treasuredata.client.TDHttpClient.call(TDHttpClient.java:532)
2023-01-31T23:36:21.549416843Z 	at com.treasuredata.client.TDClient.doGet(TDClient.java:196)
2023-01-31T23:36:21.549422302Z 	at com.treasuredata.client.TDClient.listTables(TDClient.java:443)
```

Reply:
Ohh it's actually wrapped in a TDClientProcessingException. 🤔 
This fix might not be adequate.